pkgs/tools/graphics/pikchr/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
 , stdenv
 , fetchurl
+, tcl # optional, pass null to disable
```

Comment:
This is not recommended. Use this instead:

```suggestion
, enableTcl ? true, tcl
```

Reply:
Since when that changed? I remember this pattern being used in quite a few places

Reply:
Changed

Reply:
This is not hard to find instances of "remove `? null`" among pull requests. #207724 is a typical example.

On the other hand, `enableFeatures` was always standard.